components/core/Dockerfile

```diff
@@ -1,21 +1,60 @@
-FROM python:3
+# Container as boiler plate
+FROM python:3.7-slim as base 
```

Comment:
3.7 here and it's 3.6 in travis

Reply:
Shall we update Travis to use python:3.7-slim?

> 3.7 here and it's 3.6 in Travis

Reply:
I have addressed in this [PR](#754 )